modules/tracker/models/base/ScalarModelBase.php

```diff
@@ -32,6 +32,7 @@ public function __construct()
             'scalar_id' => array('type' => MIDAS_DATA),
             'trend_id' => array('type' => MIDAS_DATA),
             'user_id' => array('type' => MIDAS_DATA),
```

Comment:
In this case, the `user_id` if a submission has a user id, but I would be very happy with the other too.